src/js/components/List/List.js

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import { compose } from 'recompose';
+import styled, { css, withTheme } from 'styled-components';
+
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { InfiniteScroll } from '../InfiniteScroll';
+import { Keyboard } from '../Keyboard';
+import { Text } from '../Text';
+import { backgroundStyle, focusStyle, normalizeColor } from '../../utils';
+import { withFocus, withForwardRef } from '../hocs';
+
+const StyledList = styled(Box)`
+  list-style: none;
```

Comment:
making sure you are aware of this https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#Accessibility_concerns